### PR TITLE
feat: add sorted-set examples

### PR DESCRIPTION
**Pull request by rishtigupta**
## PR Description:
- Update the composer.json to reference the `main` branch of `client-sdk-php` repo since we currently do not have a released sdk version that contains the sorted-set apis
- Add sorted-set examples

## Issue
https://github.com/momentohq/client-sdk-php/issues/213
https://github.com/momentohq/dev-eco-issue-tracker/issues/1033

### CODE REVIEW

File: examples/composer.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "require": {
-    "momentohq/client-sdk-php": "1.11.1",
+    "momentohq/client-sdk-php": "dev-main",
```
Comment (malandis) [inline examples/composer.json:3]: 🎗️ to revert this change once the new SDK is released

**Review by malandis — COMMENTED** [review]
Lgtm!

File: examples/sorted-set-example.php [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+if ($response->asSuccess()) {
+    $logger->info("SUCCESS: Added elements to set\n");
+} elseif ($err = $response->asError()) {
+    $logger->info("Error adding elements to set: {$err->message()}\n");
+    exit(1);
+}
+
+// fetch sorted set by rank (0 to -1 for all elements) in ascending order
+$logger->info("Fetching sorted set $SET_NAME\n");
+$response = $client->sortedSetFetchByRank($CACHE_NAME, $SET_NAME, 0, -1);
```
Comment (nand4011) [inline examples/sorted-set-example.php:75]: Why not leave off the ranks for fetching all elements?

Comment (rishtigupta) [reply examples/sorted-set-example.php:75]: I am looking into that. It was giving me error saying:
```
[ex:]: Error fetching sorted set: Invalid argument passed to Momento client: minScore must be less than or equal to maxScore
```

**Review by nand4011 — APPROVED** [review]
I think we should show fetching in descending order using a named argument if possible, LGTM overall. We should get this in.
